base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -326,12 +315,42 @@ impl ConsensusConstants {
             max_block_transaction_weight: 19500,
             median_timestamp_count: 11,
             emission_initial: 10_000_000.into(),
-            emission_decay: 0.999,
-            emission_decay_int: &EMISSION_DECAY,
+            emission_decay: &EMISSION_DECAY,
+            emission_tail: 100.into(),
+            max_randomx_seed_height: std::u64::MAX,
+            proof_of_work: algos,
+            faucet_value: 0 * T,
+        }]
+    }
+
+    pub fn ridcully() -> Vec<Self> {
+        let difficulty_block_window = 90;
+        let mut algos = HashMap::new();
+        // seting sha3/moneor to 40/60 split
+        algos.insert(PowAlgorithm::Sha3, PowAlgorithmConstants {
+            max_target_time: 1800,
+            min_difficulty: 60_000_000.into(),
+            target_time: 300,
+        });
+        algos.insert(PowAlgorithm::Monero, PowAlgorithmConstants {
+            max_target_time: 1200,
+            min_difficulty: 59_000.into(),
+            target_time: 200,
```

Comment:
Should this not be 240  * 0.4?
```suggestion
            target_time: 96,
```

Reply:
@hansieodendaal might be able to provide this figure

Reply:
Nope
Its 120*0.4. Its the chain target time (120 second) * block percentage. 


If you want a 50-50 split you do chain target time(120) * 50% = 120*0.5 = 240 as the target time. 
If you want a 30-30-30 split you do chain target time(120) * 30% = 120*0.3 = 360 as the target time.

Reply:
`200` looks correct
See my first comment